installing/installing_bare_metal/installing-bare-metal.adoc

```diff
@@ -54,6 +54,16 @@ include::modules/csr-management.adoc[leveloffset=+2]
 * See xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installation-three-node-cluster_installing-bare-metal[Configuring a three-node cluster] for details about deploying three-node clusters in bare metal environments.
 * See xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installation-approve-csrs_installing-bare-metal[Approving the certificate signing requests for your machines] for more information about approving cluster certificate signing requests after installation.
 
+[id="installation-requirements-baremetal-vsphere_{context}"]
+== Requirements for baremetal clusters on vSphere
+
+Ensure you enable the `disk.EnableUUID` parameter on all virtual machines in your cluster.
+
+[role="_additional-resources"]
+.Additional resources
+
+* See xref:../../installing/installing_vsphere/installing-vsphere.adoc#installation-vsphere-machines_installing-vsphere[Installing RHCOS and starting the OpenShift Container Platform bootstrap process] step 9f for details on setting the `disk.EnableUUID` to `TRUE` on VMware vSphere for user-provisioned infrastructure.
```

Comment:
```suggestion
* See xref:../../installing/installing_vsphere/installing-vsphere.adoc#installation-vsphere-machines_installing-vsphere[Installing RHCOS and starting the OpenShift Container Platform bootstrap process] for details on setting the `disk.EnableUUID` value to `TRUE` on VMware vSphere for user-provisioned infrastructure.
```
- For future maintainability, I would remove the hard-coded reference to a specific step.
- Add "value" to `disk.EnableUUID`; literals have to include a reference.